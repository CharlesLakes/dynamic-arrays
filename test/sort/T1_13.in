10
81 98 35 73 56 44 31 94 25 3