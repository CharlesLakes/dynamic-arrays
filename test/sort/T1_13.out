3 25 31 35 44 56 73 81 94 98